.github/workflows/release.yml

```diff
@@ -19,9 +19,17 @@ jobs:
         with:
           name: installer
 
+      - name: Download appx 📥
+        uses: actions/download-artifact@v3
+        with:
+          name: appx
+
+      - name: Rename appx ✏️
+        run: mv */*.msixbundle NixOS-WSL.msixbundle
```

Comment:
actions do not allow post run or similar?

Reply:
What do you mean by that?

Reply:
That we do the mv command after the previous and not in a new step. Not that important.

Reply:
I don't think something like that exists, and it doesn't really make a difference either way (unlike with dockerfiles for example)